src/index.test.jsx

```diff
@@ -245,4 +245,32 @@ describe('web components', () => {
 		});
 		assert.equal(getShadowHTML(), '<p>Active theme: sunny</p>');
 	});
+
+	function ManualRegistration() {
+		return <div>I'm manually registered</div>;
+	}
+
+	it('allows manual deferred registration by just exposing the element', async () => {
+		const CustomElement = registerElement.toCustomElement(
```

Comment:
```suggestion
		const CustomElement = createCustomElement(
```

will need to add an import up top obv